package.json

```diff
@@ -12,13 +12,16 @@
     "test:hardhat": "NODE_ENV=test npx hardhat test",
     "dev": "next dev -p 8080",
     "build": "next build",
+    "build:vercel": "npm run fgu:vercel && npm run build",
```

Comment:
I think it would be _slightly_ better to use GH actions to apply the DB changes. But it doesn't look like we have a GH workflow for deployments and instead use Vercel's GH default integration so this seemed like the simplest way to apply DB changes in CI for now.